.github/workflows/push-master.yml

```diff
@@ -0,0 +1,17 @@
+name: push-master
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  lint-and-test:
+    uses: ./.github/workflows/lint-and-test.yml
+
+  push-master:
+    name: Push master
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Generate coverage report 📦
+        run: npm run test:coverage:push
```

Comment:
push-master doesn't tell anything. Maybe add-badges or create-code-coverage-report?

Reply:
It's the flow which executes on push on master, can change the name to push-master-and-create-code-coverage-report.

Reply:
sounds good